hyperlane/README.md

```diff
@@ -65,9 +65,18 @@ hyperlane warp deploy --config ./configs/warp-config.yaml --registry ./registry
 ```
 go install ./cmd/hyp
 
-hyp deploy 127.0.0.1:9090
+# Deploy with NoopISM (for testing)
+hyp deploy-noopism 127.0.0.1:9090 69420
+
+# Deploy with ZK Execution ISM
+hyp deploy-zkism 127.0.0.1:9090 [evm-rpc] [ev-node-rpc] 69420
+
+# Deploy with Multisig ISM
+hyp deploy-multisigism 127.0.0.1:9090 [validators-comma-separated] [threshold] 69420
```

Comment:
Maybe it would be good to add here that the validators list should be eth-style 20 byte 0x hex addresses.